File: update_search.js
const fs = require('fs');
const path = require('path');

const dir = 'c:/Users/El Rosal/Desktop/New folder';
const files = fs.readdirSync(dir).filter(f => f.endsWith('.html'));

files.forEach(file => {
    const filePath = path.join(dir, file);
    let content = fs.readFileSync(filePath, 'utf8');
    
    content = content.replace(/oninput="searchProducts\(this\.value\)"/g, 'oninput="showSuggestions(this.value)"');
    content = content.replace(/oninput="searchProducts && searchProducts\(this\.value\)"/g, 'oninput="showSuggestions(this.value)"');
    
    fs.writeFileSync(filePath, content, 'utf8');
});

console.log(`Updated ${files.length} HTML files.`);

File: update_search.py
import os
import glob
import re

html_files = glob.glob('c:/Users/El Rosal/Desktop/New folder/*.html')
for file in html_files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    # Replace the searchProducts call in oninput with showSuggestions
    content = re.sub(r'oninput="searchProducts\(&#36;this\.value\)"', 'oninput="showSuggestions(this.value)"', content)
    content = re.sub(r'oninput="searchProducts && searchProducts\(&#36;this\.value\)"', 'oninput="showSuggestions(this.value)"', content)
    
    # Actually wait, the regex above has &#36; instead of $? No, the original is searchProducts(this.value)
    # Let me fix the regex string
    content = re.sub(r'oninput="searchProducts\(this\.value\)"', 'oninput="showSuggestions(this.value)"', content)
    content = re.sub(r'oninput="searchProducts && searchProducts\(this\.value\)"', 'oninput="showSuggestions(this.value)"', content)
    
    with open(file, 'w', encoding='utf-8') as f:
        f.write(content)

print(f"Updated {len(html_files)} HTML files.")
